app/client/components/productPage.tsx

```diff
@@ -280,36 +318,51 @@ const getProductDetailRenderer = (
               <ProductDetailRow
                 label={productPageProperties.tierRowLabel}
                 data={
-                  productPageProperties.tierChangeable ? (
-                    <div css={wrappingContainerCSS}>
-                      <div css={{ marginRight: "15px" }}>
-                        {productType.alternateTierValue || productDetail.tier}
+                  <>
+                    {productPageProperties.tierChangeable ? (
+                      <div css={wrappingContainerCSS}>
+                        <div css={{ marginRight: "15px" }}>
+                          {productType.alternateTierValue || productDetail.tier}
+                        </div>
+                        {/*TODO add a !=="Patron" condition around the Change tier button once we have a direct journey to cancellation*/}
+                        <a
+                          href={
+                            "https://membership." +
+                            window.guardian.domain +
+                            "/tier/change"
+                          }
+                        >
+                          <Button text="Change tier" right />
+                        </a>
                       </div>
-                      {/*TODO add a !=="Patron" condition around the Change tier button once we have a direct journey to cancellation*/}
-                      <a
-                        href={
-                          "https://membership." +
-                          window.guardian.domain +
-                          "/tier/change"
-                        }
-                      >
-                        <Button text="Change tier" right />
-                      </a>
-                    </div>
-                  ) : (
-                    productType.alternateTierValue || productDetail.tier
-                  )
+                    ) : (
+                      productType.alternateTierValue || productDetail.tier
+                    )}
+                    {productType.displayPlanName && (
+                      <i>
+                        {productType.displayPlanName(
+                          getMainPlan(productDetail.subscription)
+                        )}
+                      </i>
+                    )}
+                  </>
                 }
               />
             ) : (
               undefined
             )}
-            <ProductDetailRow
-              label={"Start date"}
-              data={formatDate(
-                productDetail.subscription.start || productDetail.joinDate
-              )}
-            />
+            {productDetail.joinDate !== productDetail.subscription.start && (
```

Comment:
Personally I find being shown both a join date _and_ a start date slightly confusing - is join date the date that I signed up via the website and start date the date that my first paper will arrive? Was this a new request from UX?

Reply:
Also from a language perspective, it’s a membership that you ‘join’ and a subscription that you ‘sign up for:’ and which ‘starts on:’.

Reply:
@jacobwinch 

> is join date the date that I signed up via the website and start date the date that my first paper will arrive

yes to that but no not a request from UX, it was that previously join date was used as a fallback which occasionally misleading, I figured providing both would be explicit.

Based on @paulbrown1982's point about language, I could easily have it display join date for membership and start date for other subscription... I'll do that.

Reply:
done in https://github.com/guardian/manage-frontend/pull/196/commits/f1a04a98b3241bcfaa871a43a761c6c01a3380a0